src/eei.cpp

```diff
@@ -235,13 +235,13 @@ namespace hera {
       return Literal();
     }
 
-    if (import->base == Name("getBalance")) {
+    if (import->base == Name("getExternalBalance")) {
```

Comment:
I would rather have things in line with existing naming schemes like RPC.
In that can, getExternal* would become get get*At which, in my opinion, makes more sense upon reading.

Reply:
This is in line with the EEI.

Reply:
Ok the name change should probably be introduced in a later revision, as it would probably break a lot of tests for the current one.

Maybe having multiple aliases for EEI methods would be good for backwards compatibility, at the cost of clarity